### DISCUSSION TITLE
Are there well maintained domain block lists available to reference?

### DISCUSSION THREAD

**Discussion by brookmiles** [mastodon/mastodon#18143] [Q&A] [answered]
I am in the process of setting up a mastodon instance, and would like to know if there are any which quality lists available to reference of known problematic servers that I could refer to, either to proactively block, or simply keep an eye out for future activity in order to reduce the impact on my users.

Of course who the "right" servers to block are will depend on your goals and viewpoint, but I am specifically looking for lists along the lines of those found at https://mastodon.social/about/more#unavailable-content and https://koyu.space/about/more#unavailable-content.

These lists are a good start, but because some of the names are redacted I cannot make full use of them. Does either site make an unredacted list available? I wasn't able to find such a list with some basic searches on github or google.

My aim here is to benefit from the experiences and effort of more experienced server operators. If there is a better forum or discussion group in which I could do so, please let me know!

**Comment by brookmiles** [comment, marked as answer]
I suppose that since I now maintain a block list, I should link to it here: https://gardenfence.github.io/

On the question of discussion forum for more experienced admins, the official Discord, available through Patreon (https://www.patreon.com/mastodon/membership), is a good place to collaborate and learn from the experience of other server operators (not blocklists specifically). Becoming a patron is not strictly necessary, you can also be invited by a current member, or possibly by asking [Eugen ](https://mastodon.social/@Gargron) directly. I'm not aware it being listed anywhere else that's easy to find.

**Comment by marians** [comment]
Could you please tell me how to get to that Discord server?

**Reply by brookmiles** [reply]
I'm not sure if it's available outside the Patreon https://www.patreon.com/mastodon/membership

**Reply by nightpool** [reply]
Note: to be clear, the Mastodon discord doesn't have any blocklists or anything like that

**Comment by hecko-yes** [comment]
I'm very late, but here are some uncensored lists that I know of (but do not explicitly endorse):
- https://hwl.li/toots/blocks/detailed (Excel file, thousands of entries, reasoning and severity tiers for all)
- https://vulpine.club/about/more#unavailable-content (also an outdated version with extended reasoning at https://vulpineclub.github.io/blocklist)
- https://mas.to/about/more#unavailable-content (no reasoning unfortunately)
- https://social.pixie.town/about/more#unavailable-content

**Comment by shleeable** [comment]
That detailed excel file is something...

There is a need for multiple different kind of block lists
* Bad IP Block list - Admins should be able to block ips from bad actors doing anything (mostly anti-spam)
* Bad email Block list - Admins should be able to block people from signing up (mostly anti-spam)
* Abuse instance Block list - There is general consensus on what are the bad instances (abuse, threats, unethical, terfy bullshit) and as such we might need to built multiple block lists with different coverage.

The major problem is blocklists age poorly, and the fediverse grows quickly so you need to be on your guard... and we need to build something automated. Any such thing should likely be managed by instance admins collaborating together instead of as an official thing **BUT** I'd also rather get to the point where people are blocking accounts over whole instances.

This is something I'm starting to investigate and raise with the Mastodon devs... I've got an idea about getting good coverage to stop spam, and similar methods can be applied to bad actors (bad accounts).

I assume admins on abuse instances will create their own blocklist of our "tilted woke instances".... We will see.
